pkg/endpoint/endpoint.go

```diff
@@ -2389,12 +2389,8 @@ func (e *Endpoint) identityResolutionIsObsolete(myChangeRev int) bool {
 
 // Must be called with e.Mutex NOT held.
 func (e *Endpoint) runLabelsResolver(owner Owner, myChangeRev int) {
-	// NOTE: UnconditionalLock is used here only for logging
-	e.UnconditionalLock()
-
-	newLabels := e.OpLabels.IdentityLabels()
+	newLabels := e.GetOpLabels()
```

Comment:
Note this now gets an array of strings (the model), not a `Labels` object. I haven't observed the output for the log specifically, but assume that this should still format similarly.